.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
Why did you use the file `.eslintrc.js` instead of `.eslintrc`?

Reply:
Also, I think our base `eslint` config should be moved to a repository on its own (i.e. `eslint-config-exolnet`)

Reply:
I decide to use `.eslintrc.js` since it's deprecated since about 2.5 years to use `.eslintrc` whitout an extention.

https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Reply:
We deceided to go with a `. eslintrc.json`